openhtf/io/test_record.py

```diff
@@ -31,7 +33,15 @@ class InvalidMeasurementDimensions(Exception):
 LogRecord = collections.namedtuple(
     'LogRecord', 'level logger_name source lineno timestamp_millis message')
 OutcomeDetails = collections.namedtuple(
-    'OutcomeDetails', 'code_type code description')
+    'OutcomeDetails', 'code description')
+
+
+class Outcome(Enum):
```

Comment:
whoops, should fix this to not set values to names as well.  you missed one @fahhem :P